eth_utils/encoding.py

```diff
@@ -1,6 +1,22 @@
+from typing import (
+    Any,
+)
+
+
 def int_to_big_endian(value: int) -> bytes:
     return value.to_bytes((value.bit_length() + 7) // 8 or 1, "big")
 
 
 def big_endian_to_int(value: bytes) -> int:
     return int.from_bytes(value, "big")
+
+
+def get_default_codec() -> Any:
+    from eth_abi.abi import (
```

Comment:
Internal imports might not be ideal, but maybe is a good fit for this situation. Since the utility allows overriding the `ABICodec` and this is initialized otherwise.

Any issues with this? Is there a better way to do this?

Reply:
I've not seen internal imports like that before. Is the purpose to reduce load time? Is there a measurable difference vs. importing at the top like normal?

A-OK if so, but if so, maybe a:

```python
from typing import (
    TYPE_CHECKING,
)

if TYPE_CHECKING:
    from eth_abi.abi import (
        ABICodec,
    )
```

To drop the `Any`?

Reply:
I was able to resolve the dependency issue between `eth-utils` and `eth-abi` in this way. I think imports are only needed when a utility uses `get_default_codec`, otherwise the user will provide their own.

I'll get the types declared though, good idea!